app/models/user.rb

```diff
@@ -23,17 +23,41 @@ def affiliations
     @affiliations ||= ENV['suAffiliation']
   end
 
+  def person_affiliations
+    @person_affiliations ||= ENV['unscoped-affiliation']
+  end
+
+  # If a user has proper Stanford affiliation to allow access to resources now depends
+  # on two kinds of affiliation information: the 'su_affiliated' method which returns true
+  # if there are specific affiliations recorded in the 'suAffiliation' Shibboleth attribiute
+  # as well that which is captured by 'unscoped-affiliation' which is a mapping for 'eduPersonAffiliation'.
+  # If the latter returns 'member', we can treat this person as having the correct affiliation status.
+  # Refer to https://uit.stanford.edu/service/saml/arp/edupa for more information.
+  # Note also that currently 'member' for eduPersonAffiliation has a one to one correspondence
+  # with the stanford:library-resources-eligible status.
   def stanford_affiliated?
+    su_affiliated? || person_affiliated?
+  end
+
+  def su_affiliated?
     return false if affiliations.blank?
 
     affiliations.split(';').any? do |affiliation|
       Settings.SU_AFFILIATIONS.include?(affiliation.strip)
     end
   end
 
+  def person_affiliated?
+    return false if person_affiliations.blank?
+
+    person_affiliations.split(';').any? do |person_affiliation|
+      person_affiliation.strip === Settings.UNSCOPED_AFFILIATION # rubocop:disable Style/CaseEquality
```

Comment:
If you're doing a straight-up string comparison, I think you could consider `==` instead of `===` here since the latter is typically used to gauge case equality. (In this way, Ruby and JavaScript are annoyingly different.)

Reply:
Ok, I can change that